agent/address.h

```diff
@@ -60,6 +60,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <netdb.h>
```

Comment:
Right, except `struct addrinfo` is only used in `address.c` (not `address.h`), which already conditionally includes `netdb.h`. Is `HAVE_NETDB_H` not being defined in your `config.h`?